.github/workflows/manual-deploy-testnet-l2.yml

```diff
@@ -1,27 +1,42 @@
-# Deploys an Obscuro network on Azure for Testnet
-# TO BE FILLED IN
+# Deploys an Obscuro network on Azure for Testnet and Dev Testnet
 #
-# The Obscuro network is composed of 2 obscuro nodes running on individual vms
+# The Obscuro network is composed of 2 obscuro nodes running on individual vms with SGX
 #
 
 name: '[M] Deploy Testnet L2'
 
 on:
   workflow_dispatch:
     inputs:
+      testnet_type:
+        description: 'Testnet Type'
+        required: true
+        default: 'dev-testnet'
+        type: choice
+        options:
+          - 'dev-testnet'
+          - 'testnet'
       L1HOST:
-        description: 'geth network fqdn address'
+        description: 'L1 FQDN Address (defaults to Dev Testnet L1)'
         required: true
-        default: 'testnet-gethnetwork.uksouth.azurecontainer.io'
+        default: 'dev-testnet-gethnetwork.uksouth.azurecontainer.io'
```

Comment:
`dev-testnet-gethnetwork.uksouth.azurecontainer.io`

that is specific to dev-testnet though isn't it?

Reply:
so what would it be for testnet?

Reply:
`testnet-gethnetwork.uksouth.azurecontainer.io`

But we prolly don't need the ability to specifying the L1 Host. ( We haven't really used so far).
So I'm removing this input.